config.dev.toml

```diff
@@ -320,6 +320,15 @@ IndexType = "none"
         # Password is the password for a redis SingleFlight lock.
         # Env override: ATHENS_REDIS_PASSWORD
         Password = ""
+
+        [SingleFlight.Redis.LockConfig]
+            # TTL for the lock in seconds. Defaults to 300 seconds (5 minutes).
+            TTL = 300
+            # Timeout for acquiring the lock in seconds. Defaults to 300 seconds (5 minutes).
+            Timeout = 300
+            # Max retries while acquiring the lock. Defaults to 300.
+            MaxRetries = 300
```

Comment:
should this be 5?

Reply:
It's currently set to 300 IINW so I used the same as the default value: https://github.com/gomods/athens/blob/7d67ee76ce1e70d281c7ffef2423679801b7aa38/pkg/stash/with_redis.go#L48

Reply:
Hmm! This does not seem right, Should we revisit these values and have saner defaults. A request can hang for ever.
cc @marwan-at-work

Reply:
Updated to 10.